test/emmy/structure_test.cljc

```diff
@@ -542,6 +557,20 @@
                           (s/down 'g 'h)))))
         "sumr uses g/+, so symbols etc can be added too."))
 
+  (testing "fold-chain"
```

Comment:
I like it. I can easily imagine wanting a level-cutoff feature, but that's only because I've been in numpy-land recently and that way of thinking is fresh on my mind, so please don't do it :)